db/migrate/20151015190030_create_sponsor_group.rb

```diff
@@ -0,0 +1,17 @@
+class CreateSponsorGroup < ActiveRecord::Migration
+  def change
+    create_table :sponsor_groups do |t|
+      t.string :name
+      t.text :address
+      t.string :phone
+      t.string :email
+      t.string :citizenship_status
+      t.string :refugee_connection
```

Comment:
We'll have to figure out what we want to do about this. ie I think it needs a boolean that says if they have a connection + the relationship + if they're outside country of origin.